README.md

```diff
@@ -35,21 +35,21 @@ These steps are to be completed by the author of the blog post.
     
       Copy the [post-single-author.adoc](./templates/post-single-author.adoc) file (or [post-multiple-authors.adoc](./templates/post-multiple-authors.adoc) for multiple authors) to the [posts](./posts) directory and rename the file using the format `YYYY-MM-DD-post-title.adoc`, where the date represents the expected publication date (e.g. `2021-11-21-open-liberty-is-awesome.adoc`).
 
-      Place any images in [img/blog](./img/blog/).
+      Place any images in [img/blog](./img/blog/). Add a [border to all screenshots](#troubleshooting-asciidoc) to neaten the edges.
```

Comment:
There might be some cases where the author's screenshots have a border. Would it be better if we say "If your picture lacks a border, see [border to all screenshots](#troubleshooting-asciidoc) to neaten the edges."? 

Or should be recommend that authors avoid adding screenshots with borders and rely on the website to provide the border? If we ask the author to rely on the website's `.img_border_light` CSS classname, this could avoid having screenshots with different border colors, though most likely everyone will use a dark color like black.

Reply:
I think in general people don't put borders around screenshots, but I'm fine with saying that. Diagrams etc are less of a problem - I don't really care what colour border they use. This is less about having a visible border (ideally you can't even really see the light grey border line) but more about sharpening the edges where the edges otherwise look ragged.

Reply:
Actually, I'm going to leave this instruction as it is. If the screenshot has a dark border already, they should just use the `.img_border_dark` class name. If it's a screenshot (rather than a diagram), the edges are probably not that neat, so this makes the edge more neatly defined.

Reply:
I've removed "all" so it's less prescriptive. The tag is in the template anyway (which I'd forgotten when I wrote this - I was looking for the tag in the readme when reviewing someone's post that hadn't used it).